ebook.ddoc

```diff
@@ -56,4 +56,7 @@ NOT_EBOOK=
 RUNNABLE_EXAMPLE=$0
 RUNNABLE_EXAMPLE_STDIN=
 RUNNABLE_EXAMPLE_ARGS=
+SPEC_SUBNAV_NEXT=
+SPEC_SUBNAV_PREV_NEXT=
+SPEC_SUBNAV_PREV=
```

Comment:
That's why there are a few changes on the diff. By putting `dlangspec.tex` into `web` before running `test`, the undefined macros were detected.